debezium-connector-jdbc/src/test/java/io/debezium/connector/jdbc/e2e/AbstractJdbcSinkIT.java

```diff
@@ -175,6 +181,10 @@ protected List<SinkRecord> consumeSinkRecords(int numRecords) {
                 });
 
         sinkTask.put(records);
+        if (sinkTask.getLastProcessingException() != null) {
+            throw new RuntimeException("JDBC sink throw an exception processing the data", sinkTask.getLastProcessingException());
```

Comment:
Just a nitpick if `RuntimeException` is used here then method signature can stay the same.